From 1bf273d347087421c828284fdf40a6a2ee21ebdd Mon Sep 17 00:00:00 2001
From: onedots <onedots0@outlook.com>
Date: Sat, 4 Jul 2026 07:58:36 -0700
Subject: [PATCH 5/5] dts: rename onedots artifact namespace to quartzix

Renames the board dts/dtsi/-u-boot.dtsi files and fixes the Makefile
dtb entry, #include, model/compatible strings to match. Verified:
'make DEVICE_TREE=... dtbs' output (including the auto-included
-u-boot.dtsi content) recompiles with only the expected model/
compatible string changes, byte-diffed against the pre-rename build.
---
 arch/arm/dts/Makefile                                       | 2 +-
 ...u-boot.dtsi => stm32mp157c-quartzix-512d-v1-u-boot.dtsi} | 0
 ...onedots-512d-v1.dts => stm32mp157c-quartzix-512d-v1.dts} | 6 +++---
 .../{stm32mp15xx-onedots.dtsi => stm32mp15xx-quartzix.dtsi} | 0
 4 files changed, 4 insertions(+), 4 deletions(-)
 rename arch/arm/dts/{stm32mp157c-onedots-512d-v1-u-boot.dtsi => stm32mp157c-quartzix-512d-v1-u-boot.dtsi} (100%)
 rename arch/arm/dts/{stm32mp157c-onedots-512d-v1.dts => stm32mp157c-quartzix-512d-v1.dts} (98%)
 rename arch/arm/dts/{stm32mp15xx-onedots.dtsi => stm32mp15xx-quartzix.dtsi} (100%)

diff --git a/arch/arm/dts/Makefile b/arch/arm/dts/Makefile
index f169133e..e2e4504b 100644
--- a/arch/arm/dts/Makefile
+++ b/arch/arm/dts/Makefile
@@ -838,7 +838,7 @@ dtb-$(CONFIG_STM32MP15x) += \
 	stm32mp157f-ed1.dtb \
 	stm32mp157f-ev1.dtb \
 	stm32mp15xx-dhcom-pdk2.dtb \
-	stm32mp157c-onedots-512d-v1.dtb \
+	stm32mp157c-quartzix-512d-v1.dtb \
 	stm32mp157d-atk.dtb
 
 dtb-$(CONFIG_SOC_K3_AM6) += k3-am654-base-board.dtb k3-am654-r5-base-board.dtb
diff --git a/arch/arm/dts/stm32mp157c-onedots-512d-v1-u-boot.dtsi b/arch/arm/dts/stm32mp157c-quartzix-512d-v1-u-boot.dtsi
similarity index 100%
rename from arch/arm/dts/stm32mp157c-onedots-512d-v1-u-boot.dtsi
rename to arch/arm/dts/stm32mp157c-quartzix-512d-v1-u-boot.dtsi
diff --git a/arch/arm/dts/stm32mp157c-onedots-512d-v1.dts b/arch/arm/dts/stm32mp157c-quartzix-512d-v1.dts
similarity index 98%
rename from arch/arm/dts/stm32mp157c-onedots-512d-v1.dts
rename to arch/arm/dts/stm32mp157c-quartzix-512d-v1.dts
index e449f52d..d0412ad5 100644
--- a/arch/arm/dts/stm32mp157c-onedots-512d-v1.dts
+++ b/arch/arm/dts/stm32mp157c-quartzix-512d-v1.dts
@@ -10,11 +10,11 @@
 #include "stm32mp15xd.dtsi"
 #include "stm32mp157-pinctrl.dtsi"
 #include "stm32mp15xxac-pinctrl.dtsi"
-#include "stm32mp15xx-onedots.dtsi"
+#include "stm32mp15xx-quartzix.dtsi"
 
 / {
-	model = "OneDots STM32MP157-512D PRO V1 Board";
-	compatible = "st,stm32mp157c-onedots-512d-v1", "st,stm32mp157";
+	model = "Quartzix STM32MP157-512D PRO V1 Board";
+	compatible = "st,stm32mp157c-quartzix-512d-v1", "st,stm32mp157";
 
 	aliases {
 		ethernet0 = &ethernet0;
diff --git a/arch/arm/dts/stm32mp15xx-onedots.dtsi b/arch/arm/dts/stm32mp15xx-quartzix.dtsi
similarity index 100%
rename from arch/arm/dts/stm32mp15xx-onedots.dtsi
rename to arch/arm/dts/stm32mp15xx-quartzix.dtsi
-- 
2.43.0

